src/sidebar/util/test/annotation-metadata-test.js

```diff
@@ -234,6 +234,58 @@ describe('annotation-metadata', function() {
     });
   });
 
+  describe('.isHighlight', () => {
```

Comment:
I presume the period here was following the convention of other functions in this file. Usually however the `describe` block for a standalone function is just `describe('functionName')`.

Reply:
Exactly. Didn't wanna deal with fixing the whole file at present.